docs/installations/installation-on-macos.md

```diff
@@ -52,8 +53,8 @@ Make sure `<OpenMS-PATH>` points to the folder where OpenMS is installed locally
    cd /Applications/OpenMS-<version>
    sudo xattr -r -d com.apple.quarantine *
    ```
-2. Bug with running Java based thirdparty tools like MSGFPlusAdapter and LuciphorAdapter from within TOPPAS.app
-   If you face issues while running Java based thirdparty tools from within `TOPPAS.app`, run the TOPPAS.app from within
-   the Terminal.app (e.g. with the `open` command) to get access to the `path` where Java is located.
-   Java is usually present in the `PATH` of the terminal. Adavanced users can set this path in the `Info.plist` of/inside
-   the `TOPPAS.app`.
+2. Bug with running Java based thirdparty tools like MSGFPlusAdapter and LuciphorAdapter from within **TOPPAS.app**
+   If you face issues while running Java based thirdparty tools from within TOPPAS.app, run the TOPPAS.app from within
```

Comment:
can you add a newline b/t 56 and 57 line number.

Reply:
done